ssjetpackcomposeprogressbutton/src/main/java/com/simform/ssjetpackcomposeprogressbuttonlibrary/SSJetPackComposeProgressButton.kt

```diff
@@ -135,6 +142,17 @@ fun SSJetPackComposeProgressButton(
     var failureAlphaValue by remember { mutableStateOf(zeroFloat) }
     var progressAlphaValue by remember { mutableStateOf(zeroFloat) }
     var cornerRadiusValue by remember { mutableStateOf(cornerRadius) }
+    val borderColor by animateColorAsState(
+        targetValue = (
+                if (buttonState == SSButtonState.LOADING) animatedButtonBorderColor
+                else buttonBroderColor
+                ) ?: Color.Transparent,
```

Comment:
Keep it like this for readability:

if (buttonState == SSButtonState.LOADING)
                    animatedButtonBorderColor
                else
                    buttonBroderColor
                ) ?: Color.Transparent,